spacecmd/src/spacecmd/system.py

```diff
@@ -4606,6 +4606,8 @@ def help_system_scheduleproductmigration(self):
 \n    Options: \
 \n        -s START_TIME \
 \n        -d pass this flag, if you want to do a dry run \
+\n        -v pass this flag if you want to allow vendor change \
```

Comment:
`-v` is not always used consistently in `spacecmd` as a short-hand for `--verbose` but quite often that's now it is used. Many other tools also use `-v` for the same purpose. I would prefer if we could name the option for allowing vendor change differently.

Reply:
Thank you for the review, do you think something like `-avc` would be better than `-v`? For this parameter I have a hard time thinking of a single letter that makes sense other than `v`.

When I added this shortcut, I didn't care too much about these possible overlaps because we've already used `-d` as a shortcut to `--dry-run` for example and it's typically used to enable debug mode.

Reply:
I'm not sure if `-avc` is possible, it might be interpreted as `-a -v -c`. For me, there just is no single letter that makes sense. In `zypper` there is no short flag either. Do we need to have a single letter flag?

> When I added this shortcut, I didn't care too much about these possible overlaps because we've already used `-d` as a shortcut to `--dry-run` for example and it's typically used to enable debug mode.

`-d` is also used inconsistently in `spacecmd`, that's true. We can't easily (or quickly) change that, but we can pay more attention when we add new `-d`/`-v`flags.